release_notes/40-release-notes.adoc

```diff
@@ -190,6 +191,10 @@ Without this improvement, Sensor reprocesses Kubernetes and {osp}-specific event
 
 - Instructions for adding integrations with Microsoft Azure have been added. See xref:../operating/manage-user-access/configuring-identity-providers/connecting-azure-ad-to-rhacs-using-sso-configuration.adoc#user-authentication[Connecting Azure AD to {product-title-short} using SSO configuration].
 
+- (Release 4.0.1) Service description documentation that provides an overview of {rh-rhacscs-first}, including its features, capabilities, and architecture have been added. See xref:../cloud_service/rhacs-cloud-service-service-description.adoc#rhacs-cloud-service-service-description[{product-title-managed-short} service description].
```

Comment:
Maybe we should use "Version 4.0.1" to be consistent with the rest of the doc (rather than release). We kind of use interchangeably but mostly seem to use "version" when naming a specific version.